certified-connectors/Certopus/readme.md

```diff
@@ -0,0 +1,43 @@
+# Certopus
```

Comment:
@VrajGohil please include all required sections in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

Reply:
I've updated the readme in accordance with template.